src/main/java/com/fakng/fakngagrgtr/parser/amazon/AmazonParser.java

```diff
@@ -71,6 +72,7 @@ private Vacancy createVacancy(VacancyDTO dto) {
         Vacancy vacancy = new Vacancy();
         vacancy.setTitle(dto.getTitle());
         vacancy.setUrl(parseUrl(dto));
+        vacancy.setJobId(dto.getIdIcims());
```

Comment:
I think we should use dto.getId() here.

Reply:
Well... We can
But in fact for Google id field looks like this: job/123456
Whereas idIcims: 123456

We can save the full one or just number. I thought that storing number is simpler and less mem consuming (shorter string, hehe). And you can always add constant prefix "job/", if required

Do you think the full id would be better?

Reply:
No. Let's keep as it.

Reply:
> Well... We can But in fact for Google id field looks like this: job/123456 Whereas idIcims: 123456
> 
> We can save the full one or just number. I thought that storing number is simpler and less mem consuming (shorter string, hehe). And you can always add constant prefix "job/", if required
> 
> Do you think the full id would be better?

Its better because there is no costs to get job id from db and make further dos. Instead you should keep in head or found it from somewhere that saved job id should have prefix 'job/'.